src/VecSim/algorithms/brute_force/vector_block.h

```diff
@@ -34,6 +50,11 @@ struct VectorBlock {
         this->members[index] = member;
     }
 
+    std::vector<std::pair<float, labelType>> ComputeScores(DISTFUNC<float> DistFunc,
```

Comment:
```suggestion
    std::vector<std::pair<float, labelType>> computeBlockScores(DISTFUNC<float> DistFunc,
```
keep camelCase please